src/lib/plugin.js

```diff
@@ -28,6 +28,9 @@ function * requestRetry (opts, errorMessage, credentials) {
         ca: credentials.ca,
         json: true
       }, lodash.isUndefined)))
+      if (res.statusCode > 400 && res.statusCode < 500) {
```

Comment:
So with this change we retry on client errors too? Is there a use case for this?